pkg/probe/probe.go

```diff
@@ -0,0 +1,57 @@
+// (C) Copyright Confidential Containers Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package probe
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+var logger = log.New(log.Writer(), "[probe/probe] ", log.LstdFlags|log.Lmsgprefix)
+var readizProbesDone bool
+var checker Checker
+
+const CC_RUNTIMECLASS_NAME string = "kata-remote"
+
+func StartupHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function could be simpified imo

```go
func StartupHandler(w http.ResponseWriter, r *http.Request) {
	if !readizProbesDone {
		readizProbesDone, err := checker.AllDedicatedPeerPodsAreReady()
		if err != nil || !readizProbesDone {
			logger.Printf("Not all PeerPods ready, because %s", err)
			w.WriteHeader(http.StatusInternalServerError)
			return
		}
	}
	logger.Printf("All PeerPods standup. we do not check the PeerPods status any more.")
	w.WriteHeader(http.StatusOK)
}
```